bucket/eclipse-mat.json

```
@@ -1,12 +1,12 @@
 {
-    "version": "1.15.0.20231206",
+    "version": "1.16.1.20250109",
     "description": "Eclipse Memory Analyzer",
-    "homepage": "https://www.eclipse.org/mat/",
-    "license": "EPL-1.0",
+    "homepage": "https://eclipse.dev/mat/",
+    "license": "EPL-2.0",
```

```suggestion
    "license": "EPL-2.0",
    "suggest": {
        "JDK": [
            "java/openjdk17",
            "java/openjdk"
        ]
    },
```
Add Java suggestion based on https://eclipse.dev/mat/download/#:~:text=The%20minimum%20Java%20version%20required%20to%20run%20the%20stand%2Dalone%20version%20of%20Memory%20Analyzer%20is%20Java%2017.%20See%20JRE/JDK%20Sources.

---

Yes, done it all! and a new pr [6445](https://github.com/ScoopInstaller/Scoop/pull/6445)